packages/mana/consensusbasevector.go

```diff
@@ -187,7 +187,7 @@ func (c *ConsensusBaseManaVector) GetManaMap() (NodeMap, error) {
 	defer c.Unlock()
 	res := make(map[identity.ID]float64)
 	for ID := range c.vector {
-		mana, err := c.getMana(ID)
+		mana, err := c.getMana(ID, false)
```

Comment:
`GetManaMap()` function in mana plugin also calls `ConsensusBaseManaVector.GetManaMap()`. Since higher level functions in the mana plugin all return current values, we should add ` update ...bool` to `ConsensusBaseManaVector.GetManaMap()` and propagate `update` to this `getMana` call, instead of passing `false`.
Therefore, the higher level `manaPlugin.GetManaMap()` can get the mana map with updated values, and `getPastConsensusManaVectorHandler` can get the non updated values.